pages/blog/may-2019-update.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Here are the most significant changes in May.
+---
+
+# May 2019 Update
+
+**Olivier Tassinari**
+
+*June 8, 2019*
+
+Here are the most significant changes in May:
+
+- 🎉 We have released Material-UI v4 stable during the React Europe conference.
+You can find the details in our [blog post 📝](https://medium.com/material-ui/material-ui-v4-is-out-4b7587d1e701) and a quick introduction in this [5 minutes talk](https://www.youtube.com/watch?v=m54_CPfbWow).
+- 🐛 We have made two consequent releases after v4.0.0, containing a significant amount of fixes (v4.0.1 and v4.0.2).
+
+But this summary is just scratching the surface. We have accepted 206 commits from 78 different contributors. We have changed 3,031 files with 96,616 additions and 85,768 deletions.
+
+## Our roadmap intent for June
+
+*(We'll do our best, no guarantee!)*
+
+- We will release an improved Slider component with range support. You can already [preview it](https://deploy-preview-15703--material-ui.netlify.com/components/slider/).
+
+![Slider](/static/blog/may-2019-update/slider.png)
+
+- We will release a new Tree View component. You can already [preview it](https://deploy-preview-14827--material-ui.netlify.com/components/tree-view/).
```

Comment:
The pressure! 😆

Reply:
I will help, we have 30 days :)

> (We'll do our best, no guarantee!)

Would you be more comfortable with "We will work on a new Tree View…?

Reply:
Aiming for release in June might be a bit ambitious I'm all up for it but maybe `We will work on a new Tree View` is more realistic.

Reply:
I'm thinking of moving the TreeView into the lab first. What do you think?

Reply:
Sounds good to me